help/landing/governance-privacy-security/customer-managed-keys/overview.md

```diff
@@ -13,6 +13,10 @@ Data stored on Adobe Experience Platform is encrypted at rest using system-level
 
 This document provides a high level overview of the process for enabling the customer-managed keys (CMK) feature in Platform, and the prerequisite information required to complete these steps.
 
+>[!NOTE]
+>
+>For Customer Journey Analytics customers, please follow the instructions in the [Customer Journey Analytics documentation](https://experienceleague.adobe.com/docs/analytics-platform/using/cja-privacy/cmk.html?lang=en).
```

Comment:
Looks good, but can we remove the '?lang=en' query parameter please. Then i'm happy